### PR TITLE
Add how to use wrappers to README and refactor MetadataViews comments

### PR DESCRIPTION
**Pull request by alilloig**
Closes: #95

## Description

Updates `README.md` including new views and an explanation on how to use views featuring wrapper views

Refactors `MetadataViews.cdc` re-ordering some views and standardizing the comments to the cadecence docgen tool format

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-nft/blob/master/CONTRIBUTING.md#styleguides).
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: README.md
Comment (joshuahannan) [inline README.md:188]: It looks like some of these are missing descriptions. Can you add some for the ones that are missing?

File: README.md
Comment (joshuahannan) [inline README.md:203]:
```suggestion
When exposing a view that could have multiple occurrences on a single NFT, such as `Edition`, `Royalty`, `Media` or `Trait` the wrapper view should always be used, even if there is only a single occurrence. The wrapper view is always the plural version of the single view name and can be found below the main view definition in the `MetadataViews` contract.
```

File: README.md
Comment (joshuahannan) [inline README.md:205]:
```suggestion
When resolving the view, the wrapper view should be the returned value, instead of returning the single view or just an array of several occurrences of the view.
```

File: README.md
Comment (joshuahannan) [inline README.md:227]:
```suggestion
// `resolveView` should always return the same type that was passed to it as an argument, so this is improper usage because it returns `Edition` instead of `Editions`.
pub fun resolveView(_ view: Type): AnyStruct? {
```

File: contracts/MetadataViews.cdc
Comment (joshuahannan) [inline contracts/MetadataViews.cdc:143]:
```suggestion
    /// Optional view for collections that issue multiple objects
```

**Review by joshuahannan — COMMENTED** [review]
Thanks for making these changes! Remember to also pull the latest changes from master so there are no merge conflicts.

**Review by joshuahannan — APPROVED** [review]
Looks good! Lets wait for one more review before we merge

**Review by satyamakgec — APPROVED** [review]
LGTM

**Pull request merged**
